code/Test_definitions/connected-network-type-subscriptions.feature

```diff
@@ -1,289 +1,488 @@
+
+  @connected_network_type_subscriptions_creation_401.3_invalid_access_token
+  Scenario: Invalid access token
+    Given the header "Authorization" is set to an invalid access token
+    And use BaseUrL
+    And the request body is set to a valid request body
+    When the request "createConnectedNetworkTypeSubscription" is sent
+    Then the response header "Content-Type" is "application/json"
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @connected_network_type_subscriptions_retrieve_401.4_no_authorization_header
+  Scenario: No Authorization header
+    Given the header "Authorization" is removed
+    And use BaseUrL
+    When the request "retrieveConnectedNetworkTypeSubscription" is sent
+    Then the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @connected_network_type_subscriptions_retrieve_401.5_expired_access_token
+  Scenario: Expired access token
+    Given the header "Authorization" is set to an expired access token
+    And use BaseUrL
+    When the request "retrieveConnectedNetworkTypeSubscription" is sent
+    Then the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @connected_network_type_subscriptions_retrieve_401.6_invalid_access_token
+  Scenario: Invalid access token
+    Given the header "Authorization" is set to an invalid access token
+    And use BaseUrL
+    When the request "retrieveConnectedNetworkTypeSubscription" is sent
+    Then the response header "Content-Type" is "application/json"
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @connected_network_type_subscriptions_delete_401.7_no_authorization_header
+  Scenario: No Authorization header
+    Given the header "Authorization" is removed
+    And use BaseUrL
+    When the request "deleteConnectedNetworkTypeSubscription" is sent
+    Then the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @connected_network_type_subscriptions_delete_401.8_expired_access_token
+  Scenario: Expired access token
+    Given the header "Authorization" is set to an expired access token
+    And use BaseUrL
+    When the request "deleteConnectedNetworkTypeSubscription" is sent
+    Then the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @connected_network_type_subscriptions_delete_401.9_invalid_access_token
```

Comment:
Duplicate of @connected_network_type_subscriptions_creation_401.2_expired_access_token

Reply:
...the same here - the requests are different